cloud/deploys/bin/lib.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
```

Comment:
question (blocking): can this file be removed?

Reply:
yes